internal/cloud/awscloud/secure-instance.go

```diff
@@ -78,6 +78,11 @@ func (a *AWS) RunSecureInstance(iamProfile, keyName, cloudWatchGroup, hostname s
 		}
 	}()
 
+	previousSI, err := a.terminatePreviousSI(identity.InstanceID)
+	if err != nil {
+		logrus.Warningf("Unable to terminate previous secure instance with instance ID (%s): %v, this job is now likely to fail", previousSI, err)
```

Comment:
logging `previousSI` in this case is not really useful in all cases? 
or maybe ... (see below)

Reply:
ah right, I think we should replace it with the host/worker id maybe